docs/csharp/whats-new/csharp-10.md

```diff
@@ -225,3 +228,12 @@ These types aren't directly represented in metadata. They include annotations th
 - <xref:System.IntPtr> instead of `nint` or `unint`.
 - `string` instead of `string?`.
 - `ValueTuple<int, int>` instead of `(int X, int Y)`.
+
+## Static abstract members in interfaces
+
+> [!IMPORTANT]
+> *static abstract members in interfaces* is a preview feature. You must Add the `<EnablePreviewFeatures>True</EnablePreviewFeatures>` in your project file. See [Preview features](https://aka.ms/dotnet-warnings/preview-features) for more information. You can experiment with this feature, and the experimental libraries that use it. We will use feedback from the preview cycles to improve the feature, including possibly making changes, before its general release.
```

Comment:
```suggestion
> *static abstract members in interfaces* is a preview feature. You must add the `<EnablePreviewFeatures>True</EnablePreviewFeatures>` in your project file. See [Preview features](https://aka.ms/dotnet-warnings/preview-features) for more information. You can experiment with this feature, and the experimental libraries that use it. We will use feedback from the preview cycles to improve the feature, including possibly making changes, before its general release.
```

Reply:
> including possibly making changes

Did this mean to say "breaking changes"?

Reply:
I did mean "making changes". I should just remove that phrase.